README.md

```diff
@@ -21,3 +21,24 @@ randomly without memory instead of systematically trying them?
 
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. Add your answer to this markdown file.
+
+## Answer
+
+Finding the runtime complexity:
+At the beginning of each call of permutationSort there is a call of isSorted to check the permutation. The isSorted function has a complexity of $\Theta(n)$.
+During the first call of permutationSort, the primary for loop runs $n$ times. Within the for loop there is a swapping function that has constant complexity, a recursive call to permutationSort that will handle 1 less element, and a call of the isSorted function. Combining this we get the recurrance relation:
+
+$$\begin{equation}T(n)=\begin{cases}1, & \text{if $n<=1$} \\
+n+n^2*T(n-1)+C, & \text{if $n>1$}.\end{cases}\end{equation}$$
```

Comment:
It seems like the recurrence relation is wrong. Where n^2 comes from according to your code?

Reply:
I found the $n^2$ from the primary for loop which has a complexity of $n$, and within this for loop there is a call to the isSorted function which is just another for loop of $n$ complexity, so nested for loops of n complexity -> $n^2$.